harness/tests/tensorboard/test_base.py

```diff
@@ -15,7 +15,11 @@ def test_getting_manager_instance(tmp_path: pathlib.Path) -> None:
     checkpoint_config = {"type": "shared_fs", "host_path": HOST_PATH}
     env = test_util.get_dummy_env()
     manager = tensorboard.build(
-        env.det_cluster_id, env.det_experiment_id, env.det_trial_id, checkpoint_config
+        env.det_cluster_id,
```

Comment:
These are auto-formatted

Reply:
Yup, when it gets too long `black` forces them into one-arg-per-line.